src/Areas/Startups/Services/StartupsService.cs

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Options;
+using Azure.ResourceManager.Storage;
+using Azure.ResourceManager.Storage.Models;
+using Azure; // WaitUntil
+using AzureMcp.Services.Azure.Subscription;
+using AzureMcp.Services.Azure.Tenant;
+using Azure.Storage.Blobs;
+using AzureMcp.Services.Azure;
+using Azure.ResourceManager.Resources;
+using Azure.Storage.Blobs.Models;
+
+namespace AzureMcp.Areas.Startups.Services
+{
+    public sealed class StartupsService(ISubscriptionService subscriptionService, ITenantService tenantService) : BaseAzureService(tenantService), IStartupsService
+    {
+        private const string PREFERRED_REGION = "westus";
+        private readonly ISubscriptionService _subscriptionService = subscriptionService;
+
+        // Guidance command
+        public Task<StartupsGuidanceInfo> GetGuidanceAsync()
+        {
+            var info = new StartupsGuidanceInfo(
+                Title: "Microsoft for Startups Guidance",
+                Description: "Microsoft for Startups is a global program that helps startups succeed with access to technology, coaching, and support. Startups receive free Azure credits, technical resources, expert guidance, and opportunities to connect with Microsoft partners and customers. Learn more and apply at the website.",
+                Link: "https://startups.microsoft.com/"
+            );
+            return Task.FromResult(info);
```

Comment:
You already create an object like this in your `StartupsGuidanceCommand` class. Is there a point in having this be a part of the service? You should check out what the `bestpractices` and `azureterraformbestpractices` commands do.